test-lib-openshift.sh

```diff
@@ -322,8 +322,19 @@ function ct_os_cluster_up() {
   fi
 
   mkdir -p ${dir}/{config,data,pv}
-  oc cluster up --host-data-dir=${dir}/data --host-config-dir=${dir}/config \
-                --host-pv-dir=${dir}/pv --use-existing-config --public-hostname=${cluster_ip}
+  case $(oc version| head -n 1) in
+    "oc v3.1"*)
+      oc cluster up --base-dir=${dir}/data --public-hostname=${cluster_ip}
```

Comment:
I'd prefer e.g. `--base-dir="$dir/data"` format of quotation, but I guess this is not a new problem, so NP.

Reply:
good idea, thx